paparazzi/src/main/java/app/cash/paparazzi/internal/interceptors/MatrixMatrixMultiplicationInterceptor.kt

```diff
@@ -16,28 +16,26 @@ object MatrixMatrixMultiplicationInterceptor {
     require(lhsOffset + 16 <= lhs.size) { "lhsOffset + 16 > lhs.length" }
     require(rhsOffset + 16 <= rhs.size) { "rhsOffset + 16 > rhs.length" }
     for (i in 0..3) {
-      val rhs_i0 = rhs[I(i, 0, rhsOffset)]
-      var ri0 = lhs[I(0, 0, lhsOffset)] * rhs_i0
-      var ri1 = lhs[I(0, 1, lhsOffset)] * rhs_i0
-      var ri2 = lhs[I(0, 2, lhsOffset)] * rhs_i0
-      var ri3 = lhs[I(0, 3, lhsOffset)] * rhs_i0
+      val rhs_i0 = rhs[index(i, 0, rhsOffset)]
+      var r_i0 = lhs[index(0, 0, lhsOffset)] * rhs_i0
+      var r_i1 = lhs[index(0, 1, lhsOffset)] * rhs_i0
+      var r_i2 = lhs[index(0, 2, lhsOffset)] * rhs_i0
+      var r_i3 = lhs[index(0, 3, lhsOffset)] * rhs_i0
       for (j in 1..3) {
-        val rhs_ij = rhs[I(i, j, rhsOffset)]
-        ri0 += lhs[I(j, 0, lhsOffset)] * rhs_ij
-        ri1 += lhs[I(j, 1, lhsOffset)] * rhs_ij
-        ri2 += lhs[I(j, 2, lhsOffset)] * rhs_ij
-        ri3 += lhs[I(j, 3, lhsOffset)] * rhs_ij
+        val rhs_ij = rhs[index(i, j, rhsOffset)]
+        r_i0 += lhs[index(j, 0, lhsOffset)] * rhs_ij
+        r_i1 += lhs[index(j, 1, lhsOffset)] * rhs_ij
+        r_i2 += lhs[index(j, 2, lhsOffset)] * rhs_ij
+        r_i3 += lhs[index(j, 3, lhsOffset)] * rhs_ij
       }
-      result[I(i, 0, resultOffset)] = ri0
-      result[I(i, 1, resultOffset)] = ri1
-      result[I(i, 2, resultOffset)] = ri2
-      result[I(i, 3, resultOffset)] = ri3
+      result[index(i, 0, resultOffset)] = r_i0
+      result[index(i, 1, resultOffset)] = r_i1
+      result[index(i, 2, resultOffset)] = r_i2
+      result[index(i, 3, resultOffset)] = r_i3
     }
   }
 
-  @Suppress("FunctionName")
-  private fun I(i: Int, j: Int, offset: Int): Int {
+  private fun index(i: Int, j: Int, offset: Int): Int =
     // #define I(_i, _j) ((_j)+ 4*(_i))
-    return offset + j + 4 * i
-  }
```

Comment:
revert all this, since you've added "ktlint:standard:property-naming" to @Suppress above intercept method?